core/src/utils/internal/scrollbars.ts

```diff
@@ -20,11 +20,17 @@ const internalRemoveScrollbars = () => {
 
 let internalRevert = noop;
 
+/**
+ * A function to remove the scrollbars on the body element. It can be reverted using the `revertScrollbars` function.
```

Comment:
` A function to remove the scrollbars on the body element. It can be reverted using the {@link revertScrollbars} function.`